lighthouse-core/test/config/config-test.js

```diff
@@ -160,7 +160,7 @@ describe('Config', () => {
             'ViewportDimensions', // from gatherer
           ],
           __internalOptionalArtifacts: [
-            'SourceMaps', // Not in the config.
+            'SourceMapsBlah', // Not in the config.
```

Comment:
This changes the failure mode of the test: instead of L179 being the important assertion, now L172 would simply throw an error if attempting to resolve an artifact that isn't provided by any gatherer. meh.

Reply:
> This changes the failure mode of the test: instead of L179 being the important assertion, now L172 would simply throw an error if attempting to resolve an artifact that isn't provided by any gatherer. meh.

I'm not sure anything has changed, since `config`'s check isn't that it can't find the gatherer on disk, it's just if `passes` contained a matching gatherer.

Maybe pick a different fake name, though? It's kind of confusing that it sounds like a modification of an existing gatherer :)

```suggestion
            'NotInTheConfig', // Not in the config.
```